contracts/lib/arbitration/IArbitrator.sol

```diff
@@ -44,4 +44,11 @@ interface IArbitrator {
     * @return feeAmount Total amount of fees that must be allowed to the recipient
     */
     function getSubscriptionFees(address _subscriber) external view returns (address recipient, ERC20 feeToken, uint256 feeAmount);
+
+    /**
+    * @dev Tell address of a module based on a given ID
+    * @param _id ID of the module being queried
+    * @return Address of the requested module
+    */
+    function getModule(bytes32 _id) external view returns (address);
```

Comment:
As mentioned [here](https://github.com/aragon/aragon-court/pull/270#discussion_r445616442), instead of adding this to the `IArbitrator` I would denote that the Agreement app relies on the AragonCourt interface instead. WDYT?

Reply:
But I don’t get it. What do you mean with “denote”? In `Agreement` settings we have an `IArbitrator`, not an `AragonCourt` object.

Reply:
I mean to state in the Agreement app that we are relying on another interface and not on `IArbitrator`. So I'd simply create a new interface inheriting from `IArbitrator` that adds that method and we use it from the Agreement app. Sth like: 


```sol
contract IAragonCourtArbitrator extends IArbitrator {
    function getModule(bytes32 _id) external view returns (address);
}
```

Reply:
And would you define `IAragonCourtArbitrator` in `aragon-court` repo too?
Not so sure about this option. While it’s true that the modules seem specific to Aragon Court implementation, not inherent to Arbitrator functionality, we need `getModule` to replace what we really need, something like `getTransactionFees` (similar to `getSubscriptionFees`), which is inherent to arbitrator functionality, but that we cannot add because we can’t upgrade the Controller.

Reply:
For the record, we discussed offline the idea of providing here the interface for the transaction fees oracle. Then `IAgreement` will rely on both, `IArbitrator` and `ITransactionFeesOracle`, besides the `ITransactionFeesOracle` implementation is set as a module in Aragon Court.